subsys/suit/platform/sdfw/src/suit_plat_invoke.c

```diff
@@ -54,17 +62,20 @@ int suit_plat_invoke(suit_component_t image_handle, struct zcbor_string *invoke_
 {
 	struct zcbor_string *component_id;
 	suit_component_type_t component_type = SUIT_COMPONENT_TYPE_UNSUPPORTED;
+	uint32_t timeout_ms = 0;
+	bool synchronous = false;
 	intptr_t run_address;
 	uint8_t cpu_id;
 	size_t size;
+	int ret;
```

Comment:
Could you please initialize? The value is always assigned anyway, but I think it's a bit cleaner